pkg/blobinfocache/default.go

```diff
@@ -74,3 +74,12 @@ func DefaultCache(sys *types.SystemContext) types.BlobInfoCache {
 	logrus.Debugf("Using SQLite blob info cache at %s", path)
 	return cache
 }
+
+// Cleanup removes the blob info cache directory.
+func Cleanup(sys *types.SystemContext) error {
+	dir, err := blobInfoCacheDir(sys, rootless.GetRootlessEUID())
+	if err != nil {
+		return err
```

Comment:
`DefaultCache` does not fail in this situation, so I don’t think it makes sense to fail here.